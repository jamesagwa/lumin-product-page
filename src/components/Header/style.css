.header {
  background-color: white;
  margin-bottom: 4rem;
}

.logo {
  flex: 1;
}

.page-head {
  width: 80%;
  margin: 0 auto;
  display: flex;
}

.page-title {
  flex: 5;
  text-align: left;
}

.title-1 {
  font-size: 2.8rem;
}

.filter-bar {
  flex: 2;
  text-align: right;
}

.filter-bar select {
  width: 200px;
  height: 40px;
  border: 1px solid #cccccc;
  margin-top: 40px;
  padding: 10px;
  padding: 10px;
}
