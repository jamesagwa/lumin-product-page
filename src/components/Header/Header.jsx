import React from "react";
import PropTypes from "prop-types";

import { NavBar } from "../NavBar/NavBar";
import "./style.css";

export function Header(props) {
  return (
    <header className="header">
      <NavBar />
      <div className="container mt-2">
        <div className="page-head">
          <div className="page-title">
            <h1 className="title-1">All Products</h1>
            <p className="mt-small">
              A 360
              <sup>o </sup>
              look at Lumin
            </p>
          </div>
          <div className="filter-bar">
            <select name="filter" className="">
              <option selected defaultValue disabled>
                Filter by
              </option>
            </select>
          </div>
        </div>
      </div>
    </header>
  );
}

Header.propTypes = {};
