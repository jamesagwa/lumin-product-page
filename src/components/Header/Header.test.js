import React from "react";

import { render } from "@testing-library/react";
import { Header } from "./Header";

describe("Header Component", () => {
  it("should render correctly", () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
