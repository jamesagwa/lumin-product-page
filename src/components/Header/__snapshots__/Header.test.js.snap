// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render correctly 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="navbar"
    >
      <div
        class="container"
      >
        <div
          class="logo"
        >
          <h1>
            LUMIN
          </h1>
        </div>
        <nav
          class="nav"
        >
          <ul
            class="nav-list left-nav"
          >
            <li>
              Shop
            </li>
            <li>
              Learn
            </li>
          </ul>
          <ul
            class="nav-list right-nav"
          >
            <li>
              Account
            </li>
            <li>
              Cart
            </li>
          </ul>
        </nav>
      </div>
    </div>
    <div
      class="container mt-2"
    >
      <div
        class="page-head"
      >
        <div
          class="page-title"
        >
          <h1
            class="title-1"
          >
            All Products
          </h1>
          <p
            class="mt-small"
          >
            A 360
            <sup>
              o 
            </sup>
            look at Lumin
          </p>
        </div>
        <div
          class="filter-bar"
        >
          <select
            class=""
            name="filter"
          >
            <option
              disabled=""
            >
              Filter by
            </option>
          </select>
        </div>
      </div>
    </div>
  </header>
</div>
`;
